### ISSUE TITLE
goxlr-utility desktop entry starts daemon instead of opening utility

### ISSUE THREAD

**Issue by Different-Name** [NixOS/nixpkgs#331319]
### Describe the bug

The goxlr-utility package changes the functionality of the desktop entry. Instead of opening the web interface, it instead starts the daemon.

### Expected behavior

The original functionality of the desktop entry should be kept, as currently, the only way to open the web interface is:

- The taskbar icon (not accessible for those not using a taskbar, though this is a rare subset :laughing:)
- Manually visiting http://localhost:14564/
- Running the goxlr-launcher command, as the desktop entry did originally

The goxlr-daemon command only starts the daemon in the background, strange for a desktop entry
Due to the nature of goxlr-utility, it makes sense to have this program start with the system in most use cases, there's also an option provided by the goxlr-utility nixos module that is enabled by default to autostart the utility

### Additional context

Here is the relevant code:
https://github.com/NixOS/nixpkgs/blob/52ec9ac3b12395ad677e8b62106f0b98c1f8569d/pkgs/tools/audio/goxlr-utility/default.nix#L53-L55

@errnoh

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by errnoh** [comment]
@Different-Name looked into this and now I remember why I changed this when I created the package.

The reason was this https://github.com/NixOS/nixpkgs/blob/50d494e5a4370394af0e09db2a2e864f146ff3b6/pkgs/build-support/make-startupitem/default.nix#L17-L35

With the original .desktop file it opened default browser with the goxlr interface automatically after each boot. Looking at it now it seems that `goxlr-utility` creates the autostart item in code here: https://github.com/GoXLR-on-Linux/goxlr-utility/blob/9753ce92c08170519216e15d851236e76d8f7dc8/daemon/src/platform/linux/autostart.rs#L36-L59 

Anyways, that's the reasoning why I originally changed it. Should be somewhat simple to fix. I'll think about what would be the cleanest way a bit and add a fix once I'm done.
